helm_helpers.go

```diff
@@ -254,12 +255,11 @@ func getReleaseChartVersion(rs releaseState) string {
 }
 
 // getNSTLSFlags returns TLS flags for a given namespace if it's deployed with TLS
-func getNSTLSFlags(namespace string) string {
-	tls := ""
+func getNSTLSFlags(namespace string) []string {
+	tls := []string{}
 	ns := s.Namespaces[namespace]
 	if tillerTLSEnabled(ns) {
-
-		tls = " --tls --tls-ca-cert " + namespace + "-ca.cert --tls-cert " + namespace + "-client.cert --tls-key " + namespace + "-client.key "
+		tls = append(tls, "--tls", "--tls-ca-cert", namespace+"-ca.cert", "--tls-cert", namespace+"-client.cert", "--tls-key", namespace+"-client.key")
```

Comment:
For consistency, I would add space between `+`

Reply:
That is how gofmt formats that kind of line. There is over 100 examples of this kind in the existing codebase.